File: concrete/src/main/java/pcb/uwu/unit/quantity/CandelaUnit.java
package pcb.uwu.unit.quantity;

import pcb.uwu.units.quantity.LuminousIntensityUnit;

import java.util.function.Function;

public class CandelaUnit extends LuminousIntensityUnit {

	public static final CandelaUnit CANDELA = new CandelaUnit();

	private CandelaUnit() {
		super("cd", "candela", "candelas",
				Function.identity(),
				Function.identity());
	}
}

File: implementation/src/main/java/pcb/uwu/unit/base/CandelaUnit.kt
package pcb.uwu.unit.base

import pcb.uwu.util.identity

object CandelaUnit
    : LuminousIntensityUnit(symbol = "cd",
                            singularName = "candela",
                            pluralName = "candelas",
                            translationToCanonical = identity(),
                            translationFromCanonical = identity())
